### ISSUE TITLE
Is there mode to this code? Looks like good start but doesn't really do anything

### ISSUE THREAD

**Issue by blaksec** [victims/victims-web-proxy#1]
Just curious if there's more to this project. I'd really like to set up wep-proxy internally so we can have periodic synch of the database to the dev network.

**Comment by abn** [comment]
@gchorny yes definitely more to work on. Happy to receive contributions and feedbacks. Let me know what features are required etc. NOTE: This was a PoC initially.
